TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Mqs/DeleteMq/Index.cshtml.cs

```diff
@@ -102,7 +172,16 @@ public override async Task OnPageHandlerExecutionAsync(PageHandlerExecutingConte
         Status = JourneyInstance!.State.Status;
         StartDate = JourneyInstance!.State.StartDate;
         EndDate = JourneyInstance!.State.EndDate;
+        EvidenceFileId = JourneyInstance!.State.EvidenceFileId;
+        EvidenceFileName = JourneyInstance!.State.EvidenceFileName;
+        EvidenceFileSizeDescription = JourneyInstance!.State.EvidenceFileSizeDescription;
 
         await next();
     }
+
+    private string GetFileSizeDescription(long fileSizeBytes)
+    {
+        var fileSizeMb = Math.Round((decimal)(fileSizeBytes / 1024 / 1024), 1);
+        return fileSizeMb > 1 ? $"{fileSizeMb}MB" : $"{Math.Round((decimal)(fileSizeBytes / 1024), 1)}KB";
```

Comment:
We could pull in Humanizer and have it do this busywork https://github.com/Humanizr/Humanizer?tab=readme-ov-file#bytesize

Reply:
I was thinking there might be something out there that did this!
Done